### PR TITLE
rectify the flag name tfNoRippleDirect

### PR DESCRIPTION
**Pull request by ckeshava**
## High Level Overview of Change

The codebase contains an incorrect flag name `tfNoDirectRipple`. This is modified to adhere to the documentation as `tfNoRippleDirect`

### Type of Change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [] Refactor (non-breaking change that only restructures code)
- [x] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

### Did you update HISTORY.md?

- [x] Yes
- [] No, this change does not impact library users

## Test Plan

The existing unit tests have been updated to use the correct flag name

### CODE REVIEW

**Review by pdp2121 — APPROVED** [review]
LGTM

**Comment by ckniffen** [comment]
Probably make a note that this is a small breaking bugfix

**Comment by ckeshava** [comment]
@ckniffen Should I include a note in the HISTORY.md file? Or are you referring to an API changelog somewhere?

**Comment by JST5000** [comment]
@ckeshava Yep that note should be in the HISTORY.md file :) 

We usually group changes in categories - the default ones we use as needed are:

## Added

## Fixed

## BREAKING


(Since this is a small breaking change, I'd use the ## BREAKING header and then in the individual note say "- Small fix in the api to use X instead of Y")

File: packages/xrpl/HISTORY.md [hunk 1]
```diff
@@ -3,6 +3,8 @@
 Subscribe to [the **xrpl-announce** mailing list](https://groups.google.com/g/xrpl-announce) for release announcements. We recommend that xrpl.js (ripple-lib) users stay up-to-date with the latest stable release.
 
 ## Unreleased
+### BREAKING CHANGES
+* Small fix in the API to use `tfNoDirectRipple` instead of `tfNoRippleDirect`
```
Comment (pdp2121) [inline packages/xrpl/HISTORY.md:7]: I think you would still want to mention it's a flag name changed for more clarity

**Review by khancode — APPROVED** [review]

**Pull request merged**